x-pack/plugins/observability_solution/observability/public/pages/alert_details/hooks/use_create_investigation.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IHttpFetchError, ResponseErrorBody } from '@kbn/core/public';
```

Comment:
You're right, this duplication is annoying. let's think in a follow up pr where we can put these